### PR TITLE
StoNCorrOnTrack in  modules with the highest values

### PR DESCRIPTION
**Pull request by rrabadan**
modules that have a S/N corrected by track angle less than 10 added in list of modules with highest values

### CODE REVIEW

**Comment by deguio** [comment]
please test

**Comment by deguio** [comment]
please submit to 80x as well

**Comment by rrabadan** [comment]
sign please

**Comment by deguio** [comment]
ciao @rrabadan 
sorry in my previous message I meant: Please submit to 81x as well.
could you please indicate the corresponding PR for 81x?
thanks,
F.

**Comment by rrabadan** [comment]
PR for 81X:   13560

**Comment by deguio** [comment]
+1

**Comment by davidlange6** [comment]
+1

File: DQM/SiStripMonitorClient/test/SiStripDQM_OfflineTkMap_Template_cfg_DB.py [hunk 1]
```diff
@@ -112,9 +112,9 @@
     cms.PSet(mapName=cms.untracked.string('NumberOfOfffTrackCluster'),TopModules=cms.untracked.bool(True)),
     cms.PSet(mapName=cms.untracked.string('NumberOfOfffTrackCluster'),mapSuffix=cms.untracked.string("_autoscale"),mapMax=cms.untracked.double(-1.)),
     cms.PSet(mapName=cms.untracked.string('NumberOfOnTrackCluster')),
-    cms.PSet(mapName=cms.untracked.string('StoNCorrOnTrack')),
+    cms.PSet(mapName=cms.untracked.string('StoNCorrOnTrack',TopModules=cms.untracked.bool(True)),
```
Comment (smuzaffar) [inline DQM/SiStripMonitorClient/test/SiStripDQM_OfflineTkMap_Template_cfg_DB.py]: auto merge job (80X changes in to 81X) failed to resolve the above? Is not there a missing ')' after 'StoNCorrOnTrack'? In 81X we have https://github.com/cms-sw/cmssw/commit/0249733705cf5550174cf84e78c5accf5a88623c#diff-da712dd5a8dd05d67efa0e78be33b175

**Pull request merged**